.mergify.yml

```diff
@@ -104,8 +105,21 @@ pull_request_rules:
       - author=mergify[bot]
       - "#check-success>0"
       - schedule=Mon-Mon 06:00-10:00[Europe/Paris]
-      #- created-at<3 days ago
+      - created-at<2 days ago
```

Comment:
Unrelated, but commented, so this will ensure PRs older than 2 days will be affected